packages/graphile-build-pg/src/plugins/PgAllRows.js

```diff
@@ -103,6 +107,21 @@ export default (async function PgAllRows(
                                 builder.setOrderIsUnique();
                               }
                             });
+                          } else if (isView(table) && !!uniqueIdAttribute) {
+                            builder.beforeLock("orderBy", () => {
+                              if (!builder.isOrderUnique(false)) {
+                                builder.data.cursorPrefix = [
+                                  "view_unique_key_asc",
+                                ];
+                                builder.orderBy(
+                                  sql.fragment`${builder.getTableAlias()}.${sql.identifier(
+                                    uniqueIdAttribute.name
+                                  )}`,
+                                  true
+                                );
+                                builder.setOrderIsUnique();
+                              }
+                            });
```

Comment:
I tweaked this so that other plugins can add unique orders to views first if they so desire - this way we only add the order if it's not already unique.